aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1529,5 +1879,15 @@ TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("_quantized::conv_transpose2d"),  QConvInt8<2, false>::run);
 }
 
+TORCH_LIBRARY_IMPL(onednn, MkldnnCPU, m) {
+  // Conv1D/2D/3D with unary postop
+  m.impl(TORCH_SELECTIVE_NAME("onednn::qconv1d_pointwise_pt2e"), QConvPT2E::run_pointwise);
```

Comment:
nit: you don't need to have pt2e in the name I think since the operator is independent from the flow